broker/broker-undelivered-messages-ui/src/test/java/org/codice/ddf/broker/ui/UndeliveredMessagesTest.java

```diff
@@ -56,9 +56,10 @@
     private static final String RETRY_MESSAGE = "retryMessage";
 
     private final ObjectName objectName = new ObjectName(
-            "org.apache.activemq.artemis:type=Broker,brokerName=\"" + SystemBaseUrl.getHost()
-                    + "\",module=" + MODULE + ",serviceType=Queue,address=\"" + ADDRESS
-                    + "\",name=\"" + ADDRESS + "\"");
+            "org.apache.activemq.artemis:broker=\"" + SystemBaseUrl.getHost()
```

Comment:
I was wondering what prompted all the changes from `module` to `queue`. It looks like Artemis changed the way they create the `ObjectName`? Was that the reason?